core/client/src/main/java/alluxio/client/file/options/CreateDirectoryOptions.java

```diff
@@ -29,6 +30,7 @@
 @NotThreadSafe
 public final class CreateDirectoryOptions {
   private boolean mAllowExists;
+  private Mode mMode;
```

Comment:
That is my initial change, but after the second thought, I think it is not proper to let the user set user/group as those two properties should not be set by the user (currently it is inferred from thrift connection)